src/app/connections/data/index.ts

```diff
@@ -66,11 +68,13 @@ export const ConnectionCollection = {
           // sniff the name to see if we are a new type of service
           const found = cluster.match(meshServiceRe)
           if (found) {
-            const [mesh, name, namespace, zone, _abbr, _port] = cluster.split('_')
+            const [mesh, name, namespace, zone, _abbr, port] = cluster.split('_')
             stats.$resourceMeta = {
               mesh,
-              name: `${name}.${namespace}`,
+              name,
```

Comment:
Should you name this displayName to not confuse between the different types of names?